conf/minifi.properties

```diff
@@ -48,7 +48,7 @@ nifi.content.repository.class.name=DatabaseContentRepository
 ## that implements CoreComponentStateManagementProvider
 ## (e.g. an instance of RocksDbPersistableKeyValueStoreService or UnorderedMapPersistableKeyValueStoreService)
 #nifi.state.management.provider.local=
-#nifi.state.manangement.provider.local.path=
+#nifi.state.management.provider.local.path=
```

Comment:
I've found 2 more appearances of this type in CONFIGURE.md we should fix those as well.

Reply:
Thanks, amended those